### PR TITLE
Update Homepage

### PR DESCRIPTION
**Pull request by ryanwooa**
- #66 
- #77 
- update website logo

### CODE REVIEW

File: .codesandbox/tasks.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  // These tasks will run in order when initializing your CodeSandbox project.
+  "setupTasks": [
+    {
+      "name": "Install Dependencies",
+      "command": "npm install"
+    }
+  ],
+
+  // These tasks can be run from CodeSandbox. Running one will open a log in the app.
```
Comment (zli213) [inline .codesandbox/tasks.json:10]: What this file is?

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```
Comment (zli213) [inline .devcontainer/devcontainer.json:1]: What is this file for?

**Review by zli213 — COMMENTED** [review]
No obvious issues

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (ryanwooa) [reply .devcontainer/devcontainer.json]: I have no clue about why these two folders appeared in my branch.....
Deleted

Comment (ryanwooa) [reply .devcontainer/devcontainer.json]: It's in the development branch; maybe it's created by Gsy. She used the  Codesandbox for development. 
You may need to delete them in the development branch.

**Review by kesesek — APPROVED** [review]

**Review by zli213 — APPROVED** [review]

**Pull request merged**